content/webapp/pages/visual-stories/[visualStoryId].tsx

```diff
@@ -18,28 +24,120 @@ import { SimplifiedServerData } from '@weco/common/server-data/types';
 import { capitalize } from '@weco/common/utils/grammar';
 import { isNotUndefined } from '@weco/common/utils/type-guards';
 import Standfirst from '@weco/common/views/slices/Standfirst';
-import { VisualStoriesDocument } from '@weco/common/prismicio-types';
+import CardGrid from '@weco/content/components/CardGrid/CardGrid';
+import Layout, { gridSize12 } from '@weco/common/views/components/Layout';
+import Space from '@weco/common/views/components/styled/Space';
+import Divider from '@weco/common/views/components/Divider/Divider';
+import { font } from '@weco/common/utils/classnames';
+import {
+  VisualStoriesDocument,
+  EventsDocumentData,
+} from '@weco/common/prismicio-types';
+import { isPast } from '@weco/common/utils/dates';
+import {
+  transformEventTimes,
+  getLastEndTime,
+} from '@weco/content/services/prismic/transformers/events';
+import { isFilledLinkToDocument } from '@weco/common/services/prismic/types';
+
+export const getOtherVisualStories = ({
+  documentId,
+  visualStories,
+}: {
+  documentId: string;
+  visualStories: VisualStoriesDocument[];
+}): VisualStoriesDocument[] => {
+  return visualStories.filter(
+    // We want to remove any visual stories that satisfy any of the following:
+    // 1) is the same visualStory that is being displayed based on its id
+    // 2) is linked to the same exhibition or event as the visualStory being displayed (we infer it is the same visual story)
```

Comment:
Could 2 happen without 1?

Reply:
yes, but I think I've worded this badly. Visual stories can show up at three different urls depending whether they are linked to an exhibition and event or neither.

We need to remove a visual stories from the list if the id in the url is the visual story id in the list, e.g. https://wellcomecollection.org/visual-stories/ZU0PSRIAACkATr2l
Or if the id in the url is the id of an exhibition or event that is linked from the visual story in the list, e.g. 
https://wellcomecollection.org/exhibitions/ZZP8BxAAALeD00jo/visual-stories

I'll have a go at rephrasing the comments

Reply:
Ohhh yes that makes sense, thank you!